### PR TITLE
t/ckeditor5/1520: Got rid of the symbol used to recognise view elements

### PR DESCRIPTION
**Pull request by oleq**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Internal: Got rid of the symbol used to recognize view elements (see ckeditor/ckeditor5#1520).

---

### Additional information

Part of https://github.com/ckeditor/ckeditor5/pull/1521

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -37,11 +34,7 @@ export const WIDGET_SELECTED_CLASS_NAME = 'ck-widget_selected';
  * @returns {Boolean}
  */
 export function isWidget( node ) {
-	if ( !node.is( 'element' ) ) {
-		return false;
-	}
-
-	return !!node.getCustomProperty( widgetSymbol );
+	return node.is( 'element' ) && node.hasClass( WIDGET_CLASS_NAME );
```
Comment (Reinmar) [inline src/utils.js]: It needs to be a container element, am I right? I don't think any other element type would work. cc @pjasiun

Comment (Reinmar) [reply src/utils.js]: If that's true, I'd add a `is( 'containerElement' )` check also inside `toWidget()`. And similar one for `toWidgetEditable()` for `is( 'editableElement' )`

**Comment by Reinmar** [comment]
Coverage decreased.

**Comment by oleq** [comment]
Replaced by another approach in ckeditor/ckeditor5#1532.

**Pull request closed without merging**